.github/workflows/pull.yml

```diff
@@ -46,397 +46,3 @@ jobs:
       build-environment: linux-jammy-py3.8-gcc11
       docker-image: ${{ needs.linux-jammy-py3_8-gcc11-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-jammy-py3_8-gcc11-build.outputs.test-matrix }}
-
-  linux-docs:
```

Comment:
q: are the jobs removed here just to run less CI on this PR?

Reply:
yes, I just want a sanity check most of the time